actionpack/test/dispatch/request_test.rb

```diff
@@ -1078,10 +1078,13 @@ class RequestParameterFilter < BaseRequestTest
       filter_words << lambda { |key, value|
         value.reverse! if key =~ /bargain/
       }
+      filter_words << lambda { |key, value, original_params|
+        value.replace("world!") if original_params["barg"]["bargain"] == "gain" && key == "hello"
+      }
 
       parameter_filter = ActionDispatch::Http::ParameterFilter.new(filter_words)
-      before_filter["barg"] = { :bargain => "gain", "blah" => "bar", "bar" => { "bargain" => { "blah" => "foo" } } }
-      after_filter["barg"]  = { :bargain => "niag", "blah" => "[FILTERED]", "bar" => { "bargain" => { "blah" => "[FILTERED]" } } }
+      before_filter["barg"] = { "bargain" => "gain", "blah" => "bar", "bar" => { "bargain" => { "blah" => "foo", "hello" => "world" } } }
```

Comment:
Why you change the symbol to string here? It was a symbol by design.

Reply:
Was it? I didn't see a reason for it to be a symbol. I'll change it back